### ISSUE TITLE
Handle API breaking change in dependency cachier

### ISSUE THREAD

**Issue by steppi** [biopragmatics/bioversions#22]
`bioversions.utils` depends on `cachier`. The optional argument `backend` has been removed in a recent version of `cachier`, causing the snippet below to break.

```python
#: A decorator for functions whose return values
#: should be cached and refreshed once per day
refresh_daily = cachier(
    stale_after=datetime.timedelta(days=1),
    backend="memory",
    cache_dir=BIOVERSIONS_HOME,
)
```

**Comment by cthoyt** [comment]
Wow that's a huge bummer. I will tag the version to <=1.5.0 and consider alternative ways of doing this.

**Comment by steppi** [comment]
Major bummer, I didn't realize this was your feature.

> Released in `v1.5.0` !!! 1st_place_medal partying_face
> 
> Congrats and great work, @cthoyt ! First major outside contribution to this project! I'm so proud. May it be the first of many (by you and/or users of the package)!

https://github.com/shaypal5/cachier/issues/6#issuecomment-734255699

It looks like they rebased it completely out of history.
